src/products/views/ProductCreate/handlers.ts

```diff
@@ -132,10 +133,17 @@ export function createHandler(
     };
 
     const result = await productCreate(productVariables);
+
+    // When 400 is thrown Apollo for some reason adds ApolloError in result.errors
+    // This code is required to prevent navigation in exit form
+    if (result.errors && (result.errors as any)?.networkError) {
+      errors = [...errors, ...getAllErrorMessages(result.errors as any)];
+    }
```

Comment:
I'm pretty sure http 400 is not an error should be handling in form, and especially not only in this one.